### PR TITLE
Tag CovarianceMatrices.jl v0.3.1

### PR DESCRIPTION
**Pull request by attobot**
Repository: [gragusa/CovarianceMatrices.jl](https://github.com/gragusa/CovarianceMatrices.jl)
Release: [v0.3.1](https://github.com/gragusa/CovarianceMatrices.jl/releases/tag/v0.3.1)
Travis: [Travis Build Status](https://travis-ci.org/gragusa/CovarianceMatrices.jl/builds/235929235)
Diff: [vs v0.2.2](https://github.com/gragusa/CovarianceMatrices.jl/compare/5500d2770a4f6fc26ec5d052ff6ab507e3d0c807...59edd21a8b1c47cc15425e6e737aaf567339f2bc)
`requires` vs v0.2.2: 
```diff
--- 0.2.2/requires
+++ 0.3.1/requires
@@ -1,4 +1,4 @@
-julia 0.5
+julia 0.6-
 StatsBase 
 GLM 0.6
 DataFrames
```
cc: @gragusa

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds are up to date.

### CODE REVIEW

File: CovarianceMatrices/versions/0.3.1/requires [hunk 1]
```diff
@@ -0,0 +1,4 @@
+julia 0.6-
+StatsBase 
+GLM 0.6
```
Comment (tkelman) [inline CovarianceMatrices/versions/0.3.1/requires]: sorry I missed this bit, needs 0.7 minimum for the `@formula` syntax

File: CovarianceMatrices/versions/0.3.1/requires [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+DataFrames
```
Comment (tkelman) [inline CovarianceMatrices/versions/0.3.1/requires]: (past comment wasn't quite correct) needs 0.9 minimum for the `@formula` syntax

Comment (gragusa) [inline CovarianceMatrices/versions/0.3.1/requires]: @tkelman I merged your PR. Delete the old release and created the new one (below). `attobot` correctly picked the updates (I can see your PR in the diff vs v0.2.2).

Comment (tkelman) [inline CovarianceMatrices/versions/0.3.1/requires]: yep looks good

**Pull request merged**